scripts/dts/gen_defines.py

```diff
@@ -516,10 +516,14 @@ def write_vanilla_props(node):
             # DT_N_<node-id>_P_<prop-id>
             macro2val[macro] = val
 
+        spec = prop.spec
+        if spec.string_tokenizable:
+            as_token = prop.val_as_token
+            macro2val[macro + "_STRING_TOKEN"] = as_token
```

Comment:
Please update macros.bnf accordingly.

Reply:
> Please update macros.bnf accordingly.

My mistake; this actually needs to be added to the `Property suffixes` list at the top of devicetree.h.